tokio/src/io/util/mem.rs

```diff
@@ -2,6 +2,9 @@
 
 use crate::io::{AsyncRead, AsyncWrite, ReadBuf};
 use crate::loom::sync::Mutex;
+cfg_coop! {
```

Comment:
I know this is related to the warnings, but I don't get it. From all other places using `ready` why this is the only place that needs the features macros?

Reply:
In this file only in other `cfg_coop!` blocks is the `ready!` macro actually used.